manifests/v3.10.0/service-serving-cert-signer-controller/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: openshift-service-cert-signer
+  name: service-serving-cert-signer
+  labels:
+    app: openshift-service-serving-cert-signer
+    service-serving-cert-signer: "true"
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: openshift-service-serving-cert-signer
+      service-serving-cert-signer: "true"
+  template:
+    metadata:
+      name: service-serving-cert-signer
+      labels:
+        app: openshift-service-serving-cert-signer
+        service-serving-cert-signer: "true"
+    spec:
+      serviceAccountName: service-serving-cert-signer-sa
+      containers:
+      - name: service-serving-cert-signer-controller
+        image: ${IMAGE}
+        imagePullPolicy: IfNotPresent
```

Comment:
guess this should be Always in production?

Reply:
> guess this should be Always in production?

Maybe.  Why?